Add new target FLYWOOF411 board

flywoo opened the pull request:
# Board - FLYWOOF411

This board use the STM32F411CEU6 microcontroller and have the following features:
* The 16M byte SPI flash for data logging
* USB VCP and boot select button on board(for DFU)
* Stable voltage regulation,9V/2A DCDC BEC for VTX/camera etc.And could select 5v/9v with pad
* Serial LED interface(LED_STRIP)
* VBAT/CURR/RSSI sensors input
* Suppose IRC Tramp/smart audio/FPV Camera Control/FPORT/telemetry
* Supports SBus, Spektrum1024/2048, PPM. No external inverters required (built-in).
* Supports I2C device extend(baro/compass/OLED etc)
* Supports GPS

mikeller commented on src/main/target/FLYWOOF411/target.c:
```diff
@@ -0,0 +1,44 @@
+	
+    DEF_TIM(TIM3, CH4, PB1,  TIM_USE_ANY,   0, 0), //  RSSI   1,2
+    DEF_TIM(TIM5, CH4, PA3,  TIM_USE_ANY,   0, 1), //  RX2    1,0
+    DEF_TIM(TIM2, CH3, PB10, TIM_USE_PWM,   0, 0), //  CAM CC 1,1
```
This should be `TIM_USE_ANY`.

flywoo replied:
ok, i will correction ,thank

tonyyng replied:
@flywoo:  Not sure if this is the best place to request this, but could you add a FLYWOOF411 target to iNav?

mikeller commented:
@flywoo: Now that the format for Unified Target configurations for Betaflight 4.0 has been defined, and a process to create them established, can you please open a pull request to get this target added to the `master` branch? Instructions for this can be found at https://github.com/betaflight/betaflight#new-requirements-for-the-submission-of-new-and-updated-targets.

We can do this for you, but since we do not have the actual hardware, this will mean that the target will be untested and potentially broken in 4.0, so it is preferrable if you do this, and test the resulting firmware and Unified Target configuration before submitting the pull request.

flywoo commented:
OK，i will do,thank you


--------------------------------------

Nami Wu

Product Manager

________________________________
发件人: Michael Keller <notifications@github.com>
发送时间: 2019年3月27日 7:59
收件人: betaflight/betaflight
抄送: flywoo; Mention
主题: Re: [betaflight/betaflight] Add new target FLYWOOF411 board (#7662)


@flywoo<https://github.com/flywoo>: Now that the format for Unified Target configurations for Betaflight 4.0 has been defined, and a process to create them established, can you please open a pull request to get this target added to the master branch? Instructions for this can be found at https://github.com/betaflight/betaflight#new-requirements-for-the-submission-of-new-and-updated-targets.

We can do this for you, but since we do not have the actual hardware, this will mean that the target will be untested and potentially broken in 4.0, so it is preferrable if you do this, and test the resulting firmware and Unified Target configuration before submitting the pull request.

―
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/betaflight/betaflight/pull/7662#issuecomment-476902211>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AjyyYY-nScefE9qa13GATqemI48Fmxw1ks5varRpgaJpZM4bQVx1>.

flywoo commented:
HI, Michael Keller I've added it~
[cid:ed6da1d9-76ed-434d-b366-d42228e0bc92]

--------------------------------------

Nami Wu

Product Manager

________________________________
发件人: FLY WOO
发送时间: 2019年4月1日 15:45
收件人: betaflight/betaflight; betaflight/betaflight
抄送: Mention
主题: 答复: [betaflight/betaflight] Add new target FLYWOOF411 board (#7662)

OK，i will do,thank you


--------------------------------------

Nami Wu

Product Manager

________________________________
发件人: Michael Keller <notifications@github.com>
发送时间: 2019年3月27日 7:59
收件人: betaflight/betaflight
抄送: flywoo; Mention
主题: Re: [betaflight/betaflight] Add new target FLYWOOF411 board (#7662)


@flywoo<https://github.com/flywoo>: Now that the format for Unified Target configurations for Betaflight 4.0 has been defined, and a process to create them established, can you please open a pull request to get this target added to the master branch? Instructions for this can be found at https://github.com/betaflight/betaflight#new-requirements-for-the-submission-of-new-and-updated-targets.

We can do this for you, but since we do not have the actual hardware, this will mean that the target will be untested and potentially broken in 4.0, so it is preferrable if you do this, and test the resulting firmware and Unified Target configuration before submitting the pull request.

―
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/betaflight/betaflight/pull/7662#issuecomment-476902211>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AjyyYY-nScefE9qa13GATqemI48Fmxw1ks5varRpgaJpZM4bQVx1>.

mikeller commented:
@flywoo: Unfortunately I never saw a pull request from you for this, so I have done the conversion in #7933. Please review.

flywoo commented:
Thank you very much. I checked it out，No problem

--------------------------------------

Nami Wu

Product Manager

________________________________
发件人: Michael Keller <notifications@github.com>
发送时间: 2019年4月7日 16:55
收件人: betaflight/betaflight
抄送: flywoo; Mention
主题: Re: [betaflight/betaflight] Add new target FLYWOOF411 board (#7662)


@flywoo<https://github.com/flywoo>: Unfortunately I never saw a pull request from you for this, so I have done the conversion in #7933<https://github.com/betaflight/betaflight/pull/7933>. Please review.

―
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/betaflight/betaflight/pull/7662#issuecomment-480571922>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AjyyYXRQNAitbfOZ0VRyXKsCrOpVf5jtks5vebKLgaJpZM4bQVx1>.
